packages/proteus/src/main/derived/DerivedSecrets.ts

```diff
@@ -29,7 +27,7 @@ export class DerivedSecrets {
   mac_key: MacKey;
 
   constructor() {
-    this.cipher_key = new CipherKey();
+    this.cipher_key = new CipherKey(new Uint8Array([]));
```

Comment:
Can `Uint8Array` become a mandatory property of the `constructor`?

Reply:
Yes :)